tests/OSPSuite.Core.Tests/Core/CurveChartSpecs.cs

```diff
@@ -1,20 +1,37 @@
 ﻿using System.Drawing;
 using System.Linq;
+using FakeItEasy;
 using OSPSuite.BDDHelper;
 using OSPSuite.BDDHelper.Extensions;
-using FakeItEasy;
 using OSPSuite.Core.Chart;
 using OSPSuite.Core.Domain;
 using OSPSuite.Core.Domain.Data;
 using OSPSuite.Core.Domain.UnitSystem;
+using OSPSuite.Helpers;
 
 namespace OSPSuite.Core
 {
    public abstract class concern_for_CurveChart : ContextSpecification<CurveChart>
    {
+      protected Curve _curveOnYAxis;
+      protected Curve _curveOnY2Axis;
+      protected DataRepository _obsData1;
+      protected DataRepository _obsData2;
+      protected Axis _xAxis;
+      protected Axis _yAxis;
+
       protected override void Context()
       {
-         sut = new CurveChart();
```

Comment:
It's time to get more coverage here. Removing usage of fake and creating real data